cli/install/template.go

```diff
@@ -275,6 +275,14 @@ spec:
     shortNames:
     - sp
 
+### Service Account Web ###
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: linkerd-web
+  namespace: linkerd
```

Comment:
Whoops, I realize I didn't specify this correctly in my comment on #1877. Sorry about that. We should use the `{{.Namespace}}` template variable here to make sure these are created in the correct namespace.

```
  namespace: {{.Namespace}}
```